core/src/main/java/uk/gov/justice/services/core/enveloper/Enveloper.java

```diff
@@ -37,15 +40,15 @@
 @ApplicationScoped
 public class Enveloper {
 
-    private final ObjectToJsonValueConverter objectToJsonValueConverter;
+    Clock clock;
 
-    //for CDI proxying only
-    protected Enveloper() {
-        this.objectToJsonValueConverter = null;
-    }
+    ObjectToJsonValueConverter objectToJsonValueConverter;
+
+    public Enveloper() {}
```

Comment:
Why do we have a public default constructor now? I wonder if there is a better way of supporting the EnveloperFactory